utils/defaultAnswerValidations.js

```diff
@@ -0,0 +1,22 @@
+const answerTypeMap = {
+  number: ["Currency", "Number"]
+};
+
+const validationFields = {
```

Comment:
Can we rename to `validationRuleMap` or something. fields is a little confusing given GraphQL has fields